### PR TITLE
net-im/transwhat: New package

### PR DESCRIPTION
**Pull request by ConiKost**
net-im/transwhat: New package

Closes: https://bugs.gentoo.org/652264

### CODE REVIEW

File: net-im/transwhat/transwhat-0.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	net-im/yowsup2[${PYTHON_USEDEP}]"
+DEPEND="${DEPEND}"
+
+DOCS=( "INSTALL.rst" "README.rst" "USAGE.rst" )
+
+src_prepare() {
+	default
+
+	# Fixes for Spectrum2
+	mv config.py Spectrum2/config.py
```
Comment (Zlogene) [inline net-im/transwhat/transwhat-0.2.2.ebuild]: `|| die`

File: net-im/transwhat/transwhat-0.2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	sed -i -e 's/config import SpectrumConfig/Spectrum2.config import SpectrumConfig/' transWhat/transwhat.py
```
Comment (Zlogene) [inline net-im/transwhat/transwhat-0.2.2.ebuild]: `|| die`

File: net-im/transwhat/transwhat-0.2.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A WhatsApp XMPP Gateway based on Spectrum 2 and Yowsup 2."
```
Comment (Zlogene) [inline net-im/transwhat/transwhat-0.2.2.ebuild]: no dot at the end please

File: net-im/transwhat/transwhat-0.2.2.ebuild (hunk 1 above)
Comment (ConiKost) [reply net-im/transwhat/transwhat-0.2.2.ebuild]: Fixed.

File: net-im/transwhat/transwhat-0.2.2.ebuild (hunk 2 above)
Comment (ConiKost) [reply net-im/transwhat/transwhat-0.2.2.ebuild]: Fixed.

File: net-im/transwhat/transwhat-0.2.2.ebuild (hunk 3 above)
Comment (ConiKost) [reply net-im/transwhat/transwhat-0.2.2.ebuild]: Fixed.

File: net-im/transwhat/transwhat-0.2.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+DEPEND="${RDEPEND}"
+
+DOCS=( "INSTALL.rst" "README.rst" "USAGE.rst" )
+
+src_prepare() {
+	default
+
+	# Fixes for Spectrum2
+	mv config.py Spectrum2/config.py || die
+	sed -i -e 's/config import SpectrumConfig/Spectrum2.config import SpectrumConfig/' transWhat/transwhat.py || die
```
Comment (mgorny) [inline net-im/transwhat/transwhat-0.2.2.ebuild]: Use `&` instead of repeating the first expression.

**Pull request closed without merging**